app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,87 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+/* universal */
+$parchment: #EBE5D5;
+$dark_blue: #112230;
+$light_blue: #68b7e1;
+$ocean_green: #70A58B;
```

Comment:
Color `#70A58B` should be written as `#70a58b`<br>Name of variable `ocean_green` should be written in all lowercase letters with hyphens instead of underscores